demo/package.json

```diff
@@ -7,7 +7,8 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "gulp build",
     "watch": "API_URL=http://localhost:8075/api gulp watch",
-    "start": "API_URL=http://localhost:8075/api gulp server"
+    "start": "API_URL=http://localhost:8075/api gulp server",
+    "start:prod": "NODE_ENV=production DEMO_PORT=4000 gulp server"
```

Comment:
I would have expected `API_URL` to be passed also here? Unless it's provided by docker-compose

Reply:
That API_URL is only there for the proxy, i.e. if set, the client dev server will setup a proxy so that the client can always assume the URL to the api is /api

Reply:
:+1: the docker setup is fine for me